CDTDatastore/CDTReplicator/CDTPushReplication.m

```diff
@@ -66,16 +66,9 @@ - (instancetype)copyWithZone:(NSZone *)zone
 }
 
 - (NSString *)description
-{
-    NSMutableDictionary *dictionary = [[NSMutableDictionary alloc] init];
-    
-    if ([CDTAbstractReplication validateOptionalHeaders:self.optionalHeaders error:nil]) {
-        dictionary[@"headers"] = self.optionalHeaders;
-        dictionary[@"interceptors"] = self.httpInterceptors;
-    }
-    
-    dictionary[@"target"] = TDCleanURLtoString(self.target);
-    return [NSString stringWithFormat:@"%@: %@", [self class], dictionary];
+{    
+    return [NSString stringWithFormat:@"source: %@, target: %@, headers: %@, interceptors: %@, filter: %@, query_params: %@",
```

Comment:
You should also add `[self class]` to the output so a user can tell whether it's pull or push